Orange/widgets/data/owpaintdata.py

```diff
@@ -979,9 +982,11 @@ def _check_and_set_data(data):
                     data = None
                 elif len(data.domain.attributes) > 2:
                     self.Information.use_first_two()
+            if data and data.is_sparse():
```

Comment:
Currently, if I pass sparse data with more than two attributes, I see an error `Sparse data is not supported` as well as the info message that only `Paint Data uses data from the first two attributes.` The second message is irrelevant until sparse data is supported.

I suggest we move the the check for sparsity at the very beginning. Something like:
```
        def _check_and_set_data(data):
            self.clear_messages()
            if data and data.is_sparse():
                self.Error.sparse_not_supported()
                return False
            ....
```